dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/action/ViewCMSMaintenanceAction.java

```diff
@@ -479,7 +547,339 @@ private void moveAssetsToBackupDir() throws FileNotFoundException, IOException{
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* pollschoice */
+			dc.setSQL("select * from pollschoice");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + "/Pollschoice.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* pollsdisplay */
+			dc.setSQL("select * from pollsdisplay");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + "/Pollsdisplay.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* pollsquestion */
+			dc.setSQL("select * from pollsquestion");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + "/Pollsquestion.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* pollsvote */
+			dc.setSQL("select * from pollsvote");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + "/Pollsvote.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* image */
+			_list = ImageLocalManagerUtil.getImages();
+
+			/*
+			 * The changes in this part were made for Oracle databases. Oracle has problems when
+			 * getString() method is called on a LONG field on an Oracle database. Because of this,
+			 * the object is loaded from liferay and DotConnect is not used
+			 * http://jira.dotmarketing.net/browse/DOTCMS-1911
+			 */
+
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + "/Image.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* portlet */
+
+			/*
+			 * The changes in this part were made for Oracle databases. Oracle has problems when
+			 * getString() method is called on a LONG field on an Oracle database. Because of this,
+			 * the object is loaded from liferay and DotConnect is not used
+			 * http://jira.dotmarketing.net/browse/DOTCMS-1911
+			 */
+			dc.setSQL("select * from portlet");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + "/Portlet.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				_bout.close();
+			}
+
+			/* portlet_preferences */
+
+			try{

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)